### DISCUSSION TITLE
Doubts with LIS DA with SMOPS - related to file extensions of SMOPS data

### DISCUSSION THREAD

**Discussion by srikar-iist** [NASA-LIS/LISF#1409] [Q&A]
Dear @dmocko @karsenau ,

I am learning LIS DA. For the same I have downloaded the latest SMOPS-ASCAT files from https://satepsanone.nesdis.noaa.gov/pub/product/smops/TestData/

But while i check the lislog file , the program looks particularly for "gr2" extension files instead of "nc" file which are presented in the website.
Previous version of SMOPS test data i have seen were in gr2 format: https://portal.nccs.nasa.gov/lisdata_pub/data/RS_DATA/

I have a doubt whether the assimilation is heading in correct or not since the program is running.

Please also suggest how can I change the file formats: gr2 to netcdf and vice versa. I have tried cdo for converting gr2 to netcdf but **unable to find a conversion for netcdf to gr2**.

PS : I need to edit soil moisture data before doing the LIS DA, that is the sole reason for me asking this question

**Comment by dmocko** [comment]
Hi @srikar-iist 

If you look at that website again, you will see that both `.nc` and `.gr2` format files are available:
https://satepsanone.nesdis.noaa.gov/pub/product/smops/TestData/

LIS can only read the `.gr2` files, and converting from `.nc` to `.gr2` format will not work.

Please try using the `.gr2` files directly.  This should also resolve discussion #1412 for you.

**Reply by sanjaync** [reply]
hello can you please share the link https://satepsanone.nesdis.noaa.gov/pub/product/smops/TestData/ here it has no .gr2 files
